### PR TITLE
7477.master

### PR DESCRIPTION
**Pull request by ivandudardb**
I have responded to the comments in the pull request and revised the component responsible for displaying the news teaser.
Also, the link to the node ID was removed.
The advertisement template has been edited.

### CODE REVIEW

File: web/themes/custom/customtheme/components/01-atoms/advertisement/advertisement-wide.twig
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/advertisement/advertisement-wide.twig:1]: All components using the template of block content.

**Review by HeorhiiBoliuk — COMMENTED** [review]
Something with components files. You are adding to a component file a node data information from the site

File: config/default/block.block.customtheme_authorblock.yml [hunk 1]
```diff
@@ -27,4 +27,4 @@ visibility:
   request_path:
     id: request_path
     negate: true
-    pages: "<front>\r\n/user/*\r\n/category/*"
+    pages: "<front>\r\n/user/*\r\n/category/*\r\n/node/162"
```
Comment (HeorhiiBoliuk) [inline config/default/block.block.customtheme_authorblock.yml:30]: This string is look like you are showing this block by node id

File: config/default/block.block.customtheme_page_title.yml [hunk 2]
```diff
@@ -11,16 +11,16 @@ _core:
...
-  label: CATEGORY
+  label: category
   label_display: visible
   provider: core
 visibility:
   request_path:
     id: request_path
     negate: false
-    pages: '/category/*'
+    pages: "/category/*\r\n/node/162"
```
Comment (HeorhiiBoliuk) [inline config/default/block.block.customtheme_page_title.yml:26]: This string looking like you are showing the view in the node using his id.

File: config/default/block.block.customtheme_views_block__more_articles_block_1.yml [hunk 3]
```diff
@@ -26,4 +26,4 @@ visibility:
   request_path:
     id: request_path
     negate: true
-    pages: "/newspaper_pulses_pro\r\n/user/*\r\n/category/*"
+    pages: "/newspaper_pulses_pro\r\n/user/*\r\n/category/*\r\n/node/162"
```
Comment (HeorhiiBoliuk) [inline config/default/block.block.customtheme_views_block__more_articles_block_1.yml:29]: The same like first and second comment

File: config/default/block.block.customtheme_views_block__must_read_block_1.yml [hunk 4]
```diff
@@ -26,4 +26,4 @@ visibility:
   request_path:
     id: request_path
     negate: true
-    pages: "/user/*\r\n/category/*\r\n<front>"
+    pages: "/user/*\r\n/category/*\r\n<front>\r\n/node/162"
```
Comment (HeorhiiBoliuk) [inline config/default/block.block.customtheme_views_block__must_read_block_1.yml:29]: The same

File: web/themes/custom/customtheme/components/01-atoms/advertisement/advertisement-wide.twig [hunk 5]
```diff
@@ -0,0 +1,3 @@
+
+{{ view.field.field_dvertisement_photo.original_value }}
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/advertisement/advertisement-wide.twig:2]: Dont use views data in a component

File: web/themes/custom/customtheme/components/01-atoms/node-teaser-date/node-teaser-date.twig [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div class="custom-date">
+  {{ author_name }} - {{ node.createdtime|format_date('custom', 'F j, Y') }}
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/node-teaser-date/node-teaser-date.twig:2]: Dont use views data in a component

File: web/themes/custom/customtheme/components/01-atoms/node-teaser-image/node-teaser-image.twig [hunk 7]
```diff
@@ -0,0 +1,3 @@
+{% if content.field_photo.0 %}
+  {{ content.field_photo.0 }}
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/node-teaser-image/node-teaser-image.twig:2]: Dont use views data in a component

File: web/themes/custom/customtheme/components/01-atoms/node-teaser-term/node-teaser-term.twig [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{% if content.field_category.0 %}
+  {{ content.field_category.0 }}
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/node-teaser-term/node-teaser-term.twig:2]: Dont use views data in a component

File: web/themes/custom/customtheme/components/01-atoms/node-teaser-text/node-teaser-text.twig [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<div class="teaser-text">
+  {{ node.field_teaser_text.value }}
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/node-teaser-text/node-teaser-text.twig:2]: Dont use views data in a component

File: web/themes/custom/customtheme/components/01-atoms/title/title.twig [hunk 10]
```diff
@@ -0,0 +1 @@
+<a href="{{ node_url }}" class="node-link">{{ node_title }}</a>
```
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/01-atoms/title/title.twig:1]: Dont use views data in a component

File: web/themes/custom/customtheme/components/02-molecules/news-teaser/news-teaser.twig
Comment (HeorhiiBoliuk) [inline web/themes/custom/customtheme/components/02-molecules/news-teaser/news-teaser.twig:1]: Dont use views data in a component

**Review by HeorhiiBoliuk — CHANGES_REQUESTED** [review]

**Review by HeorhiiBoliuk — APPROVED** [review]
All issues is fixed

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**